xml-compare/src/test/java/testdata/TestData.java

```diff
@@ -4,19 +4,81 @@
 import org.testng.annotations.DataProvider;
 import utils.AppProperties;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 @Log4j2
 public class TestData {
 
     @DataProvider(name = "getXmlFiles")
     public static Object[][] getXmlFiles() {
-
-        String goldData = AppProperties.getProperty("gold_data", null);
-        String outputData = AppProperties.getProperty("output_data", null);
-        String[] ignoreNodes = AppProperties.getProperty("ignore_nodes", "").split("([,;])");
+        String goldData = AppProperties.getProperty("task_one_gold_data", null);
+        String outputData = AppProperties.getProperty("task_one_output_data", null);
+        String[] ignoreNodes = AppProperties.getProperty("task_one_ignore_nodes", "").split("([,;])");
 
         return new Object[][]{
                 {goldData, outputData, ignoreNodes}
         };
     }
 
+    @DataProvider(name = "getXmlFilesFromGoldDataAndOutputDataFolders", parallel = true)
+    public static Object[][] getXmlFilesFromGoldDataAndOutputDataFolders() {
+        String gold_data_folder = AppProperties.getProperty("task_two_gold_data_folder");
+        String output_data_folder = AppProperties.getProperty("task_two_output_data_folder");
+        String[] ignoreNodesDefinitions = AppProperties.getProperty("task_two_ignore_nodes_definitions", "").split("([,;])");
```

Comment:
I think it is better to keep lines not longer then 100 chars. Java Code Conventions says about 80 chars. I prefer to keep it not longer than 100.

Reply:
Yeah I agree. Changed line lengths where I found it possible and appropriate.